### DISCUSSION TITLE
Local Multiplayer

### DISCUSSION THREAD

**Discussion by GeekyG4mer** [rafaelvcaetano/melonDS-android#295] [Ideas]
Can local multiplayer be implemented like it is in the my boy gba emulator. It is much more user friendly but if can't be done, I understand. I just wanted to give a idea.

**Comment by rafaelvcaetano** [comment]
I want to but it's challenging. melonDS was not really developed with with the idea of allowing multiple instances to run in the same process and changing that requires a large effort.
